packages/httpsnippet-client-api/src/index.js

```diff
@@ -139,32 +129,27 @@ module.exports = function (source, options) {
     metadata = Object.assign(metadata, queryParams);
   }
 
-  // If we have path parameters present, we should only add them in if we have an operationId as we don't want metadata
-  // to duplicate what we'll be setting the path in the snippet to.
-  if ('operationId' in operation.schema) {
-    const pathParams = getParamsInPath(operation, operation.path);
```

Comment:
Honestly really not sure why I duplicated the `getParamsInPath` work here from `oas` but `Oas.findOperation()` gives us everything we need for this work so 🤷